zookeeper/core/component.py

```diff
@@ -303,6 +436,25 @@ def wrapped_fn(instance) -> List[str]:
     component_cls.__dir__ = wrapped_fn
 
 
+def _wrap_call(component_cls: Type) -> None:
```

Comment:
It needs to be wrapped for access prevention.
> This closes https://github.com/larq/zookeeper/issues/280. The reason I've implemented this is that it would be very easy for bugs to appear if the user expects their __init__ to have run, but the component was never configured.

The call is wrapped to prevent the component from being used when it's not ready for it